recipes-security/optee-imx/optee-os_3.10.0.imx.bb

```diff
@@ -93,4 +93,4 @@ FILES:${PN}-staticdev = "/usr/include/optee/"
 RDEPENDS:${PN}-dev += "${PN}-staticdev"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-COMPATIBLE_MACHINE = "(imx|imx-boot-container)"
+COMPATIBLE_MACHINE = "(imx-nxp-bsp|imx-boot-container)"
```

Comment:
Since I see that `imx-boot-container` is dropped from all other `COMPATIBLE_MACHINE` - perhaps it can be dropped from here as well.

I never got the NXP OP-TEE working with Boot Container, and it is not packed at the moment. I'd check if that can safely be removed, and if needed - re-introduced later.

Reply:
I removed it.